### DISCUSSION TITLE
Form组件提交后在控件下方显示后端接口的错误信息

### DISCUSSION THREAD

**Discussion by chengshaohui** [ant-design/ant-design#39136] [General]
### Reproduction link

[https://ant.design/components/form-cn/#API](https://ant.design/components/form-cn/#API)

### Steps to reproduce

验证码表单校验成功，点击提交
后端报”验证码错误“

### What is expected?

把这个后端提供的”验证码错误“放在控件校验错误的位置？

### What is actually happening?

暂时没有找到这种API，需要通过validateStatus help hasFeedback 等属性组合实现，比较麻烦

| Environment | Info |
| --- | --- |
| antd | 5.0.2 |
| React | "react": "^17.0.2", |
| System | win10 |
| Browser | chrome96 |

**Comment by yoyo837** [comment]
这比较倾向于纯业务的需要, 不是组件库考虑的.
